edx_oauth_client/middleware.py

```diff
@@ -1,24 +1,46 @@
-import re
-import os.path
+import logging
+from urlparse import urljoin, urlparse
 
+import re
 from django.conf import settings
-from django.core.urlresolvers import reverse
 from django.contrib.auth import REDIRECT_FIELD_NAME, logout
 from django.shortcuts import redirect
+from django.urls import reverse
+from social_django.views import auth, NAMESPACE
+
 from edx_oauth_client.backends.generic_oauth_client import GenericOAuthBackend
+from edx_oauth_client.constants import LOCAL_URLS, API_URLS, OAUTH_PROCESS_URLS
```

Comment:
Please reorder imported params in alphabetical order.

Reply:
done